stories/types.stories.js

```diff
@@ -48,21 +48,44 @@ storiesOf('Types', module)
       <FormCard
         cardConfig={{
           ...props.cardConfig,
-          fieldSets: [[{
-            field: 'example',
-            type: 'objectSearchCreate',
-            endpoint: '/endpoint/',
-            createFields: [
-              { field: 'first_name', populateFromSearch: true },
-              { field: 'last_name', populateNameFromSearch: true },
-              { field: 'lawfirm', populateNameFromSearch: true },
+          fieldSets: [{
+            legend: 'Legend Text',
+            rowProps: { gutter: 16 },
+            fields: [
               {
-                field: 'organization',
+                colProps: { sm: 24, lg: 12 },
+                field: 'example',
                 type: 'objectSearchCreate',
-                createFields: [{ field: 'name' }],
+                endpoint: '/endpoint/',
+                createFields: [
+                  { field: 'first_name', populateFromSearch: true },
+                  { field: 'last_name', populateNameFromSearch: true },
+                  { field: 'lawfirm', populateNameFromSearch: true },
+                  {
+                    field: 'organization',
+                    type: 'objectSearchCreate',
+                    createFields: [{ field: 'name' }],
+                  },
+                ],
               },
-            ],
-          }]]
+              {
+                colProps: { sm: 24, lg: 12 },
+                field: 'example',
```

Comment:
Could you change this field name to something like `example 2`, for clarity's sake? That way the two fields are different